modules/oe_content_policy/README.md

```diff
@@ -0,0 +1,39 @@
+# OpenEuropa Policy Content
+
+This module provides the corporate "Policy" (oe_policy) content type.
+
+## Installation
+
+Before enabling this module, make sure the following modules are present in your codebase by adding them to your
+`composer.json` and by running `composer update`:
+
+```json
+"require": {
+    "drupal/composite_reference": "~1.0-alpha2",
+    "drupal/entity_reference_revisions": "~1.3",
+    "drupal/inline_entity_form": "~1.0-rc9",
+}
+```
+
+The `inline_entity_form` module requires the following patches to be applied:
+
+```json
+"patches": {
+    "drupal/inline_entity_form": {
+        "https://www.drupal.org/project/inline_entity_form/issues/2875716": "https://www.drupal.org/files/issues/2020-11-05/ief_removed_references_2875716-89.patch"
```

Comment:
Same as above, check that patch is correct.

Reply:
I've updated this submodule and other submodules (we had deprecated patches in all submodules).